build/charts/antrea/crds/packetcapture.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: packetcaptures.crd.antrea.io
+  labels:
+    app: antrea
+spec:
+  group: crd.antrea.io
+  versions:
+    - name: v1alpha1
```

Comment:
remember to add the new API to `docs/api.md`

Reply:
updated.